index.html

```diff
@@ -267,18 +265,18 @@ <h1>
                     </li>
                     <li>
                       <a href="https://github.com/mbonabucya">
-                        <img src="image/Vector-peace.svg" alt=" peace icon" />
+                        <img src="image/Vector-peace.svg" alt="peace icon" />
                       </a>
                   </li>
                 </ul>
-                <button class="resume">
+                <button class="resume" type="submit">
                     Get my resume
                 </button>
             </div>
             <div class="skills">
```

Comment:
There is an issue with the font in this section in your live demo.

<img width="686" alt="Screen Shot 2022-06-08 at 12 30 00" src="https://user-images.githubusercontent.com/82426682/172680260-35840719-9b84-4125-aa59-8d2638e2ca90.png">

Kindly fix it.